### PR TITLE
Honeypot feature added

### LINKED ISSUE #554: Include honeypot "hp" to prevent signups of bots on newsletter
We had some issues with bots signing up on the newsletter. A way to avoid this is to add honeypot, that sendy supports now.

```

<form action="https://freelayers.net/subscribe" method="POST" accept-charset="utf-8">
	<label for="name">Name</label><br/>
	<input type="text" name="name" id="name"/>
	<br/>
	<label for="email">Email</label><br/>
	<input type="email" name="email" id="email"/>
	<br/>
	<label for="FirstName">FirstName</label><br/>
	<input type="text" name="FirstName" id="FirstName"/>
	<br/>
	<div style="display:none;">
	<label for="hp">HP</label><br/>
	<input type="text" name="hp" id="hp"/>
	</div>
	<input type="hidden" name="list" value="A8fchTRvBw2wap1ojIfjyQ"/>
	<input type="submit" name="submit" id="submit"/>
</form>

```

Also check out more info here: The following is the subscribe form HTML code for this list. To subscribe users programmatically, use the API → https://sendy.co/api?app_path

### PR DESCRIPTION
**Pull request by abhishek71994**
Resolves #554 
Let me know if further changes are to be made.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7831,6 +7831,10 @@ <h1 class="large-h1">Subscribe to stay informed</h1>
                             <div class="col-sm-5">
                                 <input class="form-email signup-email-field" placeholder="Enter your email address" type="text" name="email" id="email">
                             </div>
+                            <div style="display:none;">
+                                <label for="hp">HP</label><br/>
+                                <input type="text" name="hp" id="hp"/>
```
Comment (duttaditya18) [inline index.html]: Don't think closing the tag is necessary.

File: js/scripts.js [hunk 2]
```diff
@@ -353,7 +353,10 @@ $(window).load(function() {
         return pattern.test(emailAddress);
         };
 
-	$('.email-subscribe').submit(function() {
+	$('.email-subscribe').submit(function(data) {
```
Comment (duttaditya18) [inline js/scripts.js]: Sendy will already check if the Honeypot field is filled or not. I don't think you need to check as well.

File: js/scripts.js [hunk 3]
```diff
@@ -353,7 +353,10 @@ $(window).load(function() {
...
+        if($('#hp').val()!==""){
+            return false;
```
Comment (duttaditya18) [inline js/scripts.js]: Sendy will already check if the Honeypot field is filled or not. I don't think you need to check as well.

**Comment by duttaditya18** [comment]
So, this should work. 👍

**Comment by abhishek71994** [comment]
Should I update according to the comments or let it be as it is as it works?

**Comment by mariobehling** [comment]
@abhishek71994 Please change it and take this out. Cheers.

**Pull request merged**